wrappers/src/stats.rs

```
@@ -102,12 +102,11 @@ pub(crate) fn set_metadata(fdw_name: &str, metadata: Option<JsonB>) {
             updated_at = timezone('utc'::text, now())",
         get_stats_table()
     );
-    Spi::run_with_args(
+    let _ = Spi::run_with_args(
         &sql,
         Some(vec![
             (PgBuiltInOids::TEXTOID.oid(), fdw_name.into_datum()),
             (PgBuiltInOids::JSONBOID.oid(), metadata.into_datum()),
         ]),
-    )
-    .unwrap();
+    );
```

Maybe we should emit a warning in the logs here instead of ignoring the error?

---

sure, fixed.